docs/_drafts/scv2-hodometer-hodometer-container.mmd

```diff
@@ -0,0 +1,28 @@
+---
+title: Hodometer container architecture
+---
+
+flowchart TD
+  H["Hodometer\n\nMetrics publisher"]
+  K["Kubernetes API"]
+  S["Core v2 Scheduler"]
+  R1["Metrics receiver"]
+  R2["Extra metrics receiver"]
+  R3["Extra metrics receiver"]
+
+  subgraph k8s-cluster[Kubernetes cluster]
+    H --> |"queries\n[gRPC]"| S
+    H --> |"queries\n[HTTP]"| K
+    H -..-> |"pushes metrics to\n[HTTP]"| R2
+  end
+  H ----> |"pushes metrics to\n[HTTP]"| R1
+  H -...-> |"pushes metrics to\n[HTTP]"| R3
+
+  classDef focusSystem      fill:#1168bd,stroke:#0b4884,color:#ffffff
+  classDef supportingSystem fill:#666,stroke:#0b4884,color:#ffffff
+  classDef consumingSystem  fill:#08427b,stroke:#052e56,color:#ffffff
+  classDef systemBoundary   fill:#eee,stroke:#ccc,stroke-width:2px,stroke-dasharray: 5 5
+
+  class H,S,K focusSystem
```

Comment:
:wrench: K and S are supporting systems, not focus systems

Reply:
Fixed in https://github.com/agrski/agrski.github.io/pull/40/commits/86476803ccb4c3acbc9cd65ff33e166b79e1007d